pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +696,22 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+			// if the configuration use only one replica we increment
+			if strings.HasPrefix(string(cr.Spec.Environment), constants.RhdmPrefix) && serverSet.JbpmCluster.EnableCluster != nil && *serverSet.JbpmCluster.EnableCluster {
+				if serverSet.Replicas == nil {
+					serverSet.Replicas = new(int32)
+					if serverSet.JbpmCluster.Replicas < 2 {
+						// in case of wrong input we set almost two
+						*serverSet.Replicas = 2
+					} else {
+						*serverSet.Replicas = serverSet.JbpmCluster.Replicas
+					}
+
+				} else {
+					*serverSet.Replicas = serverSet.JbpmCluster.Replicas
```

Comment:
probably will start up env wrongly in case `serverSet.Replicas` is set to 1 and `serverSer.JbpmCluster.Replicas` is set to one as well. Please make sure configuration like this is safe and all cases are covered by unit tests.
Also if it's possible, it would be good to add log outputs there, mainly for cases, when we are setting to replicas default count to 2.

Reply:
I'll add additional replica if the replica are set to 1 and jbpmcluster is enabled